src/FinishedList.js

```diff
@@ -0,0 +1,49 @@
+
+
+function FinishedList({location, initialState, onClickTodoItem, onDeleteTodoItem}) {
+
+    this.state = {...initialState};
+
+    this.setState = (nextState) => {
+        this.state = {
+            ...this.state,
+            ...nextState
+        }
+
+        this.render();
+    }
+
+    this.render = (finishedTodo) => {
+
+        const p = document.querySelector('.finished-todo-list-count');
```

Comment:
render context 내부에서 쓰이는 변수이긴 하나,,, 선택된 selector의 클래스 명을 봐야지만 이 변수의 용도를 알수 있다는 문제가 있네요. 이런 세세한 부분까지 신경써주시면 더욱 좋겠죠?
```suggestion
        const finishedTodoCount = document.querySelector('.finished-todo-list-count');
```